bfvmm/src/hve/arch/intel_x64/vmexit/init_signal.cpp

```diff
@@ -60,7 +60,7 @@ init_signal_handler::handle(gsl::not_null<vcpu_t *> vcpu)
         vmcs_n::guest_activity_state::wait_for_sipi
     );
 
-    return true;
+    return (m_vcpu->global_state()->init_called = true);
```

Comment:
Why did you add this back? My plan is to actually remove this variable completely from the global state as it is no longer needed (it was the sync bit)